### ISSUE TITLE
Percentage of total population infected

### ISSUE THREAD

**Issue by ccpf** [ImperialCollegeLondon/covid19model#72]
I am wondering how reliable this data on the attack rate is. Just to give some examples: 
In a report from 30 March (https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/covid-19/report-13-europe-npi-impact/) you put the numbers for Spain and Germany at 15% and 0.78%, respectively. Your latest numbers for both countries are 6.6% and 0.75% respectively (a decrease in Spain and no increase in Germany). At the same time, the German Robert Koch institute just published a report according to which about 8.6% of about half a million people having participated in an antibody study were found positive. Assuming that the study is fairly representative of the entire country, this would mean that there are about 7 million positives in Germany and that your model estimate is off by a factor of >10. 

We did some calculations of our own, and based on an Infection fatality rate of about 0.3% (average of the IFRs estimated in the Diamond Princess study (0.5%, see https://www.medrxiv.org/content/10.1101/2020.03.05.20031773v2), the German case cluster study (0.37%, see https://www.land.nrw/sites/default/files/asset/document/zwischenergebnis_covid19_case_study_gangelt_0.pdf), and the German antibody study (0.05%, see above, resulting from 8.6% or 7 million positives and 3500 deaths), we would obtain attack rates for most countries in the double digits (especially considering that deaths are under-reported).

**Comment by d-Slava** [comment]
thank you @ccpf 

could you pls share a link to Robert Koch report? (and eventually resume selection process in case the report is in German only)
Firstly, I'm not sure how reliable are cases data even for 50..59 bucket used in Verity et al's study, used in ifr.
Then it s a bit odd that neighbours counties with comparable eldery populaion, interventions and its timing has so different predicted infected %. I m affraid assumtion of same itf for all countries could be wrong, as it depends from healthcare system efficiency as well.

**Comment by ccpf** [comment]
Yes the link to the RKI report: https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/2020-04-15-en.pdf?__blob=publicationFile
It's in English. You can find those numbers on page 6, 2nd paragraph below Table 4. 
Note that I mislabeled the study, it is of course not an "antibody" study but it is the "antibiotic resistance surveillance" study, where participating labs, when doing their normal blood work on patient samples, also test for some specific pathogens as part of the national disease monitoring. SARS-CoV-2 was added to the list of pathogens to be monitored in January and since then about half a million random samples were analyzed and they found 8.9% positives (as of 14 April). Which means that the dark figure would be considerable since Germany only reported around 130,000 official positives at the time (factor 55).

**Comment by d-Slava** [comment]
thank you
it looks a bit contradictory with covid specific tests data in the same report, showing lower (7.7%) positive results. whereas one is supposed to be permormed rundomly and another on targeted population..

**Comment by ccpf** [comment]
not sure what to tell you there. It is a little surprising that the random tests yielded a higher percentage than the targeted ones. I could think of a couple of factors maybe:

1. many of the first infected in Germany were young people who caught the virus while skiing in N Italy and Austria, so if it spread mostly among this age group it could have gone undetected for a while as younger people are more likely to be asymptomatic and therefore elude targeted testing.
2. Guidelines for targeted testing may have mostly targeted people with symptoms and risk groups (older people) which until now have not been as affected in Germany as in other countries (one of the reasons for the low case fatality rate there).

But these are simply speculations. Still, in the absence of anything better I would use that number until we hear otherwise.

**Comment by gustavdelius** [comment]
@ccpf, we also arrived at the result that double-digit figures are most likely for most countries, see https://github.com/gustavdelius/covid19model/blob/master/figures/18_04_step_ifr/total_infected_2020-04-19.csv, based solely on the death data used by this model. The details are explained in the report that you can find at https://github.com/gustavdelius/covid19model/blob/master/covid19_IFR_report.pdf

**Comment by ccpf** [comment]
many thanks for those links. I just had a quick peak at the numbers and I really hope that the immunity in Spain has reached 60%. This would be great. I live in Barcelona and we've essentially been locked up for 6 weeks now with 2 1/2 more weeks to go. Please send your figures to our PM so they can ease the measures a bit. 
In a much more crude approach you could use the scaling factor from the German data (x55) and apply it to Spain which would give you about 11.2M (24%) immune as of 21 April. Coincidentally, when I tried to fit the model by Neher et al. to the number of mortalities in Spain (+50% to account for the dark figure - see for instance https://www.euromomo.eu/) the model predicts about 11.1M immune (for 21 April) which is surprisingly close to the crude factor 55 upscaling estimate. 

![image](https://user-images.githubusercontent.com/63541039/79857015-6f79a980-83cd-11ea-892c-a784316002d1.png)

**Comment by ccpf** [comment]
P.S. of course I wouldn't be surprised if the dark figure was higher in Spain than in Germany and the "true" immunity even higher.

**Comment by gustavdelius** [comment]
@ccpf, I know you were not serious when you suggested that we should send this to the Spanish PM. But even if it is not necessary, I would nevertheless like to say this again: please don't overestimate the reliability of these model predictions. It would, I believe, be irresponsible to base policy decisions on them. They should only be used as motivation to take a closer look at the immunity to estimate it in other ways. 

I think what you are doing, namely looking at what one can learn from other modelling approaches, and at looking at other data, is very valuable.

**Comment by ccpf** [comment]
Yes, I was joking of course. I have seen the uncertainties associated with your numbers so yes, very difficult to base any policy decisions on them or on any model predictions for that matter. I am actually a physicist who is just passing his time while trying to avoid cabin fever so basically just doodling. I was just surprised how the crude up-scaling came fairly close to the model. I suppose if the up-scaling factor was made to vary with time, i.e., higher in the beginning while testing was low and the dark figure higher and then decreasing as testing was ramped up, the fit to the model could be improved even more, especially since the up-scaled data are below the model predictions in February/March and start to exceed them now.
Anyway, thanks for your input and those links.

**Comment by Outlars** [comment]
Hi ccpf,
you are misreading the report from Robert-Koch-Institute. For one, the data of the 50 labs participating in the "antibiotic resistance surveillance" is included in the above numbers for all of the reporting labs in Germany. This explains why this subset can have a higher rate of positive tests. The next mistake is to assume the tests were conducted randomly - it doesn't say say so in the report, because it is not the case. Testing criteria for PCR-Tests have recommended testing mostly for people with symptoms AND contact to a confirmed case, plus health workers and hospitalised patients with pneumonia (criteria are currently being loosened to people with any respiratory symptoms, given the respective lab has free capacities). Also be aware that some people are tested more than once, so test numbers are a bit higher than number of tested individuals.
In general I recommend assuming that ones own interpretation of data is flawed most of the time, especially when it is not your field of expertise (I'm a laymen, too, by the way).

**Comment by zach-hensel** [comment]
Madrid and Catalonia have 52% of covid-19 cases and 54% of deaths and 30% of the Spanish population. If Spain has 60% of the population infected, Madrid+Catalonia have had ~104% of the population infected for this to be true. The asymptomatic rate looks to be not so high from (unless people are infected, develop immunity, and never test positive), so unless you know almost zero people in Madrid and Catalonia who did not have covid-19 symptoms in the last couple months, 60% of Spain being infected and immune is not a credible estimate imo.
